frontend/Dockerfile

```diff
@@ -5,20 +5,26 @@ WORKDIR /app
 ENV PATH /app/node_modules/.bin:$PATH
 RUN npm install -g @angular/cli
 
-COPY package.json /app/package.json
-COPY . .
+COPY src ./src
+COPY angular.json package.json package-lock.json tsconfig.app.json tsconfig.json tsconfig.spec.json ./
 
 RUN --mount=type=cache,target=/app/node_modules \
     npm install && \
     ng build --output-path=dist
 
 FROM --platform=$TARGETPLATFORM nginxinc/nginx-unprivileged:1.25-alpine
-
+ENV FRONTEND_ENV=/bemeal-frontend/env.js
+ENV FRONTEND_ENV_TEMPLATE=/env.template.js
 USER nginx
 
 COPY --chmod=0755 docker/read-environment.sh /docker-entrypoint.d/40-read-environment.sh
 COPY docker/nginx.conf /etc/nginx/conf.d/default.conf
 COPY --from=build /app/dist /bemeal-frontend
+USER root
```

Comment:
Cant do, bro

This is the runtime image, where everything belongs to root and we need to give nginx user write permissions on bemeal-frontend/env.js

Cant do that on any other stage, since its the permissions in the nginx-unprivileged we are interested in